src/Nethermind/Nethermind.Provide.Vault/KeyStore/VaultKeyStoreFacade.cs

```diff
@@ -31,7 +31,7 @@ public VaultKeyStoreFacade(IPasswordProvider passwordProvider)
 
         public string GetKey()
         {
-            var password = _passwordProvider.GetPassword();
+            var password = _passwordProvider.GetPassword(null);
```

Comment:
I don't like that use case (null) unless its default value

Reply:
I only found it after changing everything. I have some ideas to solve it but I think it is still better than:
passwordProvider.Account = address1;
pass1 = passwordProvider.GetPassword();
passwordProvider.Account = address2;
pass2 = passwordProvider.GetPassword();

Reply:
here you can pass anything and the same pass will be returned but passing Address makes no sense here
Marek made it ok here but it lead to much worse code elsewhere. I broke it here and made the code better elsewhere. In the end I would opt for this option that only looks slightly strange in one line in a plugin